src/averageAges.js

```diff
@@ -15,11 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const arrayWithYears = people
+    .filter((person) => {
+      return century ? person.sex === 'm'
+      && Math.ceil(person.died / 100) === century : person.sex === 'm';
+    })
```

Comment:
in this way, it looks more clear
```suggestion
  const arrayWithYears = people.filter(
      century 
        ? person => person.sex === 'm'
           && Math.ceil(person.died / 100) === century 
        : person => person.sex === 'm';
    )
```